### DISCUSSION TITLE
RITA beacons with a score of 1 disapear

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7885] [Unsupported Versions]
Hello!

I successfully import my RITA data in SO 2.3.120 and got also alerts for beacons with a score of 1.  

![image](https://user-images.githubusercontent.com/81571847/166212655-1d44ce50-f92f-46f5-9c0a-43576bd63ded.png)

![image](https://user-images.githubusercontent.com/81571847/166218304-2e8a7870-a5d4-45ee-bf71-f75b1811c128.png)

But why does this beacon do not show up in my custom Kibana dashboard (with or without the following filter):

![image](https://user-images.githubusercontent.com/81571847/166217349-f72c888e-30de-42d6-9aac-1d7d6f37acb0.png)

Same timeframe, no additional filters, correctly sorted, ... gone. No beacons with a score of 1.  Any ideas what I'm doing wrong?

Cheers, Ben

**Comment by weslambert** [comment]
If you review the index pattern details in Kibana, to what data type is `beacon.score` shown to be mapped?

**Reply by ben-sec** [reply]
![image](https://user-images.githubusercontent.com/81571847/166463093-1c41904e-66ef-4264-bfb2-f2a6c969fdee.png)

**Reply by weslambert** [reply]
Are you able to successfully group by `beacon.score` in Hunt?

**Reply by ben-sec** [reply]
Yes.
![image](https://user-images.githubusercontent.com/81571847/166873135-733603ec-7029-4136-b519-748ac8f2f5d5.png)

**Reply by weslambert** [reply]
To be clear, are you are able to see results containing a  `beacon.score` of `1` in the Hunt groupby?

**Comment by ben-sec** [comment]
No, but that's exactly what my issue is.

**Reply by weslambert** [reply]
I was asking because I was trying to determine if the behavior was a result of how the frontend behaved (if Kibana worked differently than SOC,etc), or if it was because of the way the data type was defined in Elasticsearch. Let me see if I can reproduce the behavior, and I'll get back with you.

**Reply by weslambert** [reply]
I am not able to reproduce this issue. I can group by `beacon.score` without issue: 

**Hunt**:
![image](https://user-images.githubusercontent.com/16829864/170281218-fd009ded-aee8-4cb2-a7f1-09b1ebe51604.png)

**Kibana**:
![image](https://user-images.githubusercontent.com/16829864/170281581-141141a1-acc6-4daf-8356-9c127f00f746.png)


Are you still having problems?

**Comment by ben-sec** [comment]
Yes, beacons with a score of 1 are gone (I only see the corresponding alerts in SOC). Weird.
